### PR TITLE
doc/rados: add PG definition

### PR DESCRIPTION
**Pull request by zdover23**
Add a definition of Placement Groups to
doc/rados/operations/placement-groups.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/rados/operations/placement-groups.rst [hunk 1]
```diff
@@ -4,9 +4,20 @@
  Placement Groups
 ==================
 
+Placement groups (PGs) are subsets of each logical Ceph pool. Placement groups
+perform the function of placing objects (as a group) into OSDs. Ceph manages
+data internally at placement-group granularity: this scales better than would
+managing individual (and therefore more numerous) RADOS objects. A cluster that
```
Comment (anthonyeleven) [inline doc/rados/operations/placement-groups.rst:10]: Suggest s/and therefore more//

File: doc/rados/operations/placement-groups.rst [hunk 2]
```diff
@@ -4,9 +4,20 @@
...
+has a larger number of placement groups (for example, 100 per OSD) is better
```
Comment (anthonyeleven) [inline doc/rados/operations/placement-groups.rst:11]: Humor me and go with 150?

**Review by anthonyeleven — APPROVED** [review]

File: doc/rados/operations/placement-groups.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/operations/placement-groups.rst:10]: You've talked me into it. Accepted.

File: doc/rados/operations/placement-groups.rst (hunk 2 above)
Comment (zdover23) [reply doc/rados/operations/placement-groups.rst:11]: I will. 

Accepted.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/55630 - Reef backport
https://github.com/ceph/ceph/pull/55631 - Quincy backport

**Pull request merged**